app/src/main/config/settings_pedro.plist

```diff
@@ -118,7 +118,7 @@
 	<string>false</string>
 	<!--  -->
 	<key>startup_request_device_owner</key>
-	<string>true</string>
+	<string>false</string>
```

Comment:
What is this change?

Reply:
If set to true, the app prompts the user to allow "Modify system settings" permission.